Assignment_1/src/service/TaxService.java

```diff
@@ -0,0 +1,40 @@
+package service;
+
+//TODO: import exception and fix exception errors
+import model.Item;
+
+public class TaxService{
+    
+    public Item calculateTax(Item item) throws InvalidItemTypeException {
+        String type = item.getType();
+        double tax;
+        switch(type) {
+            case "raw":
```

Comment:
Consider leveraging java's inheritance property to make the code cleaner and scalable.

Reply:
Used an interface in the new implementation to solve this problem